libass/ass.h

```diff
@@ -734,6 +734,24 @@ void ass_process_chunk(ASS_Track *track, const char *data, int size,
  */
 void ass_set_check_readorder(ASS_Track *track, int check_readorder);
 
+/**
+ * \brief Garbage collect events that preceed deadline.
```

Comment:
I think "garbage collection" is incorrect term here. It's deterministic and unambiguous deletion of events, so simple "Prune events" or "Delete events" would be better.